### PR TITLE
Add links to the new NEST virtual machine

### PR DESCRIPTION
**Pull request by steffengraber**
There is now a new virtual machine with NEST 2.20.0.
https://nest-simulator.org/downloads/gplreleases/NEST_2.20.0_Virtual_Box_lubuntu_18.04.ova
With Brian2, Jupyter Notebook, LFPy, MUSIC, neo, NEST, NESTML, NEURON and PyNN.

### CODE REVIEW

File: doc/download.rst [hunk 1]
```diff
@@ -25,35 +25,21 @@ If you use NEST for your project, don't forget to :doc:`cite NEST <citing-nest>`
...
-* If you are compiling NEST from source, you can |link-pre|\ |version|\ |link-post|
-
-.. |link-pre| raw:: html
-
-    <a href="https://github.com/nest/nest-simulator/archive/v
-
-.. |link-post| raw:: html
-
-    .tar.gz">download the latest release here</a>
+* If you are compiling NEST from source, you can download the latest release `here <https://github.com/nest/nest-simulator/archive/v2.20.0.tar.gz>`_.
```
Comment (jougs) [inline doc/download.rst]: I assume there was a good reason for @jessica-mitchell to introduce this more complex formatting in #1372. Can you please double-check that your simplification does not break anything?

File: doc/download.rst [hunk 2]
```diff
@@ -65,16 +51,7 @@ If you run **Windows**, this is the option for you OR if you just want to run NE
...
-
-.. |link-preova| raw:: html
-
-    <a href="https://nest-simulator.org/downloads/gplreleases/lubuntu-18.04_nest-
-
-.. |link-postova|  raw:: html
-
-     .ova">latest release of the NEST VM image</a>
-
+* Download the `latest release of the NEST VM image <https://nest-simulator.org/downloads/gplreleases/NEST_2.20.0_Virtual_Box_lubuntu_18.04.ova>`_
```
Comment (jougs) [inline doc/download.rst]: See above.

File: doc/installation/livemedia.rst [hunk 3]
```diff
@@ -47,6 +47,7 @@ Notes
 
 * To install Guest Additions, select **Devices** > **Insert Guest Additions CD image...**  (top left of the VirtualBox Window). Then, open a terminal (Ctrl+Alt+t), go to "/media/nest/VBOXADDITIONS.../" and run "sudo bash VboxLinuxAdditions.run".
 
-* To set the correct language layout for your keyboard (e.g., from "US" to "DE"), you can use the program "lxkeymap", which you start by typing "lxkeymap" in the terminal.
+* To set the correct language layout for your keyboard (e.g., from "US" to "DE"), open a terminal and type: "sudo dpkg-reconfigure keyboard-configuration". Reboot now.
```
Comment (jougs) [inline doc/installation/livemedia.rst]: "Reboot now" sounds like an order. What about "After setting the correct keyboard map you have to reboot the system to activate the changes"?

File: doc/download.rst (hunk 1 above)
Comment (steffengraber) [reply doc/download.rst]: I restored the code and changed a little bit so that the git-link works correctly.

File: doc/download.rst (hunk 2 above)
Comment (steffengraber) [reply doc/download.rst]: I restored the code.
This requires that as soon as there is a new release, a VM exists for the link to work

File: doc/installation/livemedia.rst (hunk 3 above)
Comment (steffengraber) [reply doc/installation/livemedia.rst]: I followed your suggestion.

**Pull request merged**